RELEASE.md

```diff
@@ -0,0 +1,24 @@
+# How to release a new version of the Citizens Advice Form Builder
+
+NB: We're currently using tagged releases in GitHub. In future, we'll likely be
+publishing the gem to RubyGems.
+
+## Tools
+
+I've used the 'gem-release' gem to simplify the process a little.
+Enter `gem install gem-release` to install on your local machine.
+
+## Release Process
+
+- Update `CHANGELOG.md` with any changes going into this release
+- Use `gem bump -v minor` to bump the minor release number of the gem
+- Use `gem tag` to create a git tag for the new version
+- Push the tag up to the git remote with `git push origin --tags`
+- Open a PR and get it merged into main
```

Comment:
Potentially a daft question - when we start with the whole process, should we start off main, change the changelog.md there and then the gem takes care of creating a branch? Or should we branch off first?

Reply:
I haven't really formalised the process yet, but generally I've been updating the CHANGELOG and bumping version numbers etc in whatever branch I'm currently working on.

Once that's merged to main, I add the release to the GitHub releases page.

Reply:
Makes sense, for future reference are there cases it's okay to bundle multiple changes together or would you recommend one change per release? What I'm thinking about is that in design system we often have a few PRs grouped and then run a separate release process - the process is not perfect of course.

Reply:
I figured that once the form builder is feature-complete, we'll switch to bundling up changes together. Perhaps opening a release PR off main, just for the purpose of bumping the version number and pushing the tag.